### PR TITLE
feat: Recreate type definitions for release

### PR DESCRIPTION
**Pull request by sravan-s**

### CODE REVIEW

File: scripts/index_d_ts [hunk 1]
```diff
@@ -3,1106 +3,1179 @@
...
+  export type GetDisconnect = () => Promise<void>;
+  export type GetUpdateUserInfo = (
+    nickName: string,
+    profileUrl?: string
+  ) => Promise<User>;
+  export type GetCreateGroupChannel = (channelParams: GroupChannelCreateParams) => Promise<GroupChannel>;
+  export type GetCreateOpenChannel = (channelParams: OpenChannelCreateParams) => Promise<OpenChannel>;
+  export type GetGetGroupChannel = (
+    channelUrl: string,
+    isSelected?: boolean,
```
Comment (HoonBaek) [inline scripts/index_d_ts:110]: We will add `isSelected` later

File: scripts/index_d_ts [hunk 2]
```diff
@@ -3,1106 +3,1179 @@
...
+    children?: React.ReactNode;
+    theme?: 'light' | 'dark';
+    nickname?: string;
+    profileUrl?: string;
+    dateLocale?: Locale;
+    disableUserProfile?: boolean;
+    renderUserProfile?: (props: RenderUserProfileProps) => React.ReactNode;
+    allowProfileEdit?: boolean;
+    userListQuery?(): UserListQuery;
+    config?: SendBirdProviderConfig;
```
Comment (HoonBaek) [inline scripts/index_d_ts:187]: Sendbird

**Review by HoonBaek — APPROVED** [review]
left comments

File: scripts/index_d_ts [hunk 3]
```diff
@@ -3,1106 +3,1179 @@
...
+    message: EveryMessage;
+    chainTop: boolean;
+    chainBottom: boolean;
+  }
+
+  export interface ClientUserMessage extends UserMessage, ClientMessage { }
+  export interface ClientFileMessage extends FileMessage, ClientMessage { }
+  export interface ClientAdminMessage extends AdminMessage, ClientMessage { }
+  export type EveryMessage = ClientUserMessage | ClientFileMessage | ClientAdminMessage;
+  export type ClientSentMessages = ClientUserMessage | ClientFileMessage;
```
[the remark this answers is not in the document]
Comment (sravan-s) [reply scripts/index_d_ts:94]: yeah sure

File: scripts/index_d_ts (hunk 1 above)
Comment (sravan-s) [reply scripts/index_d_ts:110]: okok

File: scripts/index_d_ts [hunk 4]
```diff
@@ -3,1106 +3,1179 @@
...
-  onCancel: () => void;
-}
+  export interface OpenChannelProps extends OpenChannelProviderProps, OpenChannelUIProps {
+  }
 
-type CustomUser = SendBird.User & {
-  isMuted: boolean;
-  role: string;
-}
+  export type OpenchannelMessageListProps = {
```
[the remark this answers is not in the document]
Comment (sravan-s) [reply scripts/index_d_ts:637]: not a good idea last minute :(

[the remark this answers is not in the document]
Comment (HoonBaek) [reply scripts/index_d_ts:637]: ok ok

**Pull request merged**